### DISCUSSION TITLE
Housing Related.

### DISCUSSION THREAD

**Discussion by akhillbj** [snigi-gupta/AttendingUniversityAtBuffalo#12] [General]
In the link provided regarding the visa interview experience and few other things. Including housing. 
All the housing are near the campus ? Or away from it?
 Where did you stay? During your grad time.!
And what according to you, in list of housing is the best to stay in terms of cost effectiveness and safety.

**Comment by Anu-sha-K** [comment]
In addition to the above question, I'd like to ask for some tips and caveats that we should know when we are looking at houses. Apologize for such a long discussion but haven't been able to get this info in detail so far.

1. For example, what are some questions we should def ask the landlord before we sign the agreement. Are there any hidden charges? Minimum utilities we should expect to be arranged for, so that we can get started easily when we arrive? Duration of the agreement? If I only stay for fall and spring semesters, I would not want to pay for summer, esp if my internship is outside NY. 
2. I looked at Disha's files (Thank you, btw. You're a goddess, for doing that), she mentioned some amazing points such as looking at the location of the apartment from the main road etc. How far away from the campus can we look at the houses? A ballpark number should help us, we don't want to end up underestimating the distance and the time it takes for us to reach our classes.
3. For those who are still awaiting their UB Person number, is there a way for us to start searching for flats? I cannot login to the UB Campus living or even facebook groups, until I receive my UB person number. So I'm worried about not being able to find a good place.
4. How is on-campus housing different from off-campus, apart from price and location? From all the conversations so far, I have heard that off-campus are much cost-friendly, so I have seen people consider this an option without exploring the on-campus at all. So I wanted to understand if there are any other benefits or disadvantages that we should know?

**Reply by snigi-gupta** [reply]
@akhillbj,

Let me explain this from the beginning. UB is present in two parts in the town. There is a north campus and there is a south campus. 

North campus: This is where your classes will be if you are studying engineering/management/law etc. This is the main campus. 
South campus: This is where dental/bio-med etc students study. 

Housing at the north-campus is super expensive. I have hardly seen an MS student staying there. The house rents start from $700 excluding utilities. So this is not a cost-effective solution for most students (Indian). 

Housing at south-campus is cost-effective and this where most of the Indian students live. 

Travel: Buses ply from the Main Circle (first/last stop on south-campus) to Flint-loop (first/last stop on North campus). You only need your student id to board this bus. You have already paid for this service in the tuition fees. So reaching north campus is convenient from the south campus.

If you do want to live near the North campus then Triads is an option. This is an old motel turned into an apartment for students. 

My opinion:
Personally, I find South campus far better than Triads. Triads is expensive $400-$500 per room and that too in sharing at times. Many also turn the living room into a hall and live. The kitchen is small and the washer dryer is coin-operated. The place looks super depressing to me. The only advantage is that the campus is 5 mins away via bus. Note, it is not walkable distance. 
Why do people live in Triads then? Personal choice. I think people who don't want distractions prefer that place and people who don't like to commute. That being said, I feel life is more social at south campus.

Most people I know (and I know many people) live at south. Why? Cheaper and bigger rooms and houses. Furnished houses. Triads doesn't have furnished houses. backyard and porch. More places to eat nearby. Local stores to get mid-night snacks etc. Easy to socialize with peers and friends. Since the south campus is 10 mins walking distance, you can use the library, canteen, gym of the south campus and that is pretty good. The UB bus-stops is 5-15 mins walking distance depending which street you choose to live in and in which block. Increase that time by 3-4 mins during winters and snow.

Do your research, and choose what is best for you

North Campus: Triads
South Campus: Springvile, Callodine, Englewood, Heath, Merrimac, Tyler, Custer (these are all street names where you should look for houses).

How to choose the house? 
Go on Google maps, put the bus stop, put the house number and street, see the walkable distance in walk mode. Ideal time should be anything between 5-18 mins.

![image](https://user-images.githubusercontent.com/56351901/112711592-fb2e6c80-8e9f-11eb-9c9a-e1e108806d1a.png)

You want to stay in the first/second/at max third block of any of the streets I mentioned. 

For Callodine and Springville the bus stop is different. You will use the Maynard bus stop.
![image](https://user-images.githubusercontent.com/56351901/112711634-40529e80-8ea0-11eb-9586-577403d315a1.png)

In between Main Circle and Maynard there is another bus-stop (Goodyear) and it stop right in front of the UB apartments, again, this is a pricey accommodation and usually undergrads stay here.

**Reply by snigi-gupta** [reply]
@Anu-sha-K 

1. For example, what are some questions we should def ask the landlord before we sign the agreement. Are there any hidden charges? Minimum utilities we should expect to be arranged for, so that we can get started easily when we arrive? Duration of the agreement? If I only stay for fall and spring semesters, I would not want to pay for summer, esp if my internship is outside NY.
Ask these questions from the landlord:
- is there a cupboard, bedframe and mattress?
- is floor carpeted? If yes whole house or just rooms?
- is microwave included? 
- is washer-dryer included? Is it coin operated? 
- is snow shoveling included?
- is grass cutting included? 
- is chair-table included? 
- will hall have a sofa, center table? 
- ask a detailed version of the furniture been given
- ask for current tenants who live there, get their feedback. 
- don't just rely on pictures. My friends have been shown pretty pictures and the house was not matching the expectations that were set
- get reviews of the house on GISA. 
- ask for how many baths? 1 bath means it will have a pot, wash-basin and a bathtub. 0.5 bath means it will also have pot and basin, but no bathtub, instead it will have a shower area. Some don't have the shower area so confirm. 

*if you have questions ask them from the landlord, you will sign the lease for 12 months, you want to make sure you have a good relationship with the owner*

Utilities are based on how many people are living and you divide accordingly. I live in a 4bhk 1.5 bath house (this house structure is rare). My internet bill is $51 per month so $12.75/person per month. Gas is $10-$15 in summers so is the electricity bill (at times $20, if you waste electricity) and in winters the gas bill $20-$40. Some landlords don't pay for water bill and that is again $10-$15. All estimates are per month.

When you arrive ask the landlord to start the electricity, gas, and water connection. The wifi can be setup in a day, so that is not an issue. You can choose between Spectrum and Verizon.

When you leave for an internship you sublease your room. That means someone else will stay in your room for that time being and pay you the rent. You pay that rent to the landlord. Usually people sublease the room a few dollars less. You post for sublease on GISA.

2. I looked at Disha's files (Thank you, btw. You're a goddess, for doing that), she mentioned some amazing points such as looking at the location of the apartment from the main road etc. How far away from the campus can we look at the houses? A ballpark number should help us, we don't want to end up underestimating the distance and the time it takes for us to reach our classes. *Previously answered*

3. For those who are still awaiting their UB Person number, is there a way for us to start searching for flats? I cannot login to the UB Campus living or even facebook groups, until I receive my UB person number. So I'm worried about not being able to find a good place.
*Did you check my GitHub repo thoroughly? There are contacts and information mentioned in one of the files. Also, Google it!  and [this](https://www.ubrents.com/).

4. How is on-campus housing different from off-campus, apart from price and location? From all the conversations so far, I have heard that off-campus are much cost-friendly, so I have seen people consider this an option without exploring the on-campus at all. So I wanted to understand if there are any other benefits or disadvantages that we should know? *Previously answered*

*Kevin, Prasad, Mo are some of the good landlords.*

**Reply by crazysal** [reply]
This is a biased opinion. 

north has better houses on the average. living conditions are better. 
costs come out to be similar. since apartments with $500 rent come with utilities like electric, wifi included. as compared to south you'll have $300 rent and find out in winter u have faulty heating and gas bill is $200 . 
yes u get individuval houses in south. but also headache of landlords. many of whom wont give back deposits, headache of getting utils paying etc bills. 
some landlords are good yes. Most though will try to take advantage of international students. Have shitty houses, not do repairs, skimp on heating/insulating houses. 
Truth is Engelwood, Merrimac, Heath are the buffalo slum streets. 
You realise that when you buy a car and are not constrained to live near bus stop. 
Most landlords know this, and also that indian grad students come in for a year or two and then leave. So nobody warns them. Or cares enough. And 'compromise' attitude of indians, we end up paying to live on sub-standard conditions. 

But yes south is more happening. More stores, eat out places etc. 

I believe in last 5 years of all the experiences people have shared, they are always biased towards where they themselves stayed. 

I'd ask people who actually stayed in both to get more opinions.

**Reply by raunaqjain** [reply]
Your opinions seem more biased with words like "buffalo slum streets", "won't get deposits", and so on. Those are pretty strong words saying Heath, Englewood and other streets as slums...

Shitty landlords and poor houses can be present anywhere. That's why one should do their research before committing - Ask on UBGisa, discuss with seniors, get reviews, etc.

Rent in South is $300-340 and as previously said:
> My internet bill is $51 per month, so $12.75/person per month. Gas is $10-$15 in summers so is the electricity bill (at times $20, if you waste electricity) and in winters the gas bill $20-$40. Some landlords don't pay for water bill and that is again $10-$15. All estimates are per month.

These are per person and therefore it comes to approximately $400-$450 (in winters) at max. You won't be paying $200 of gas individually.

For North (basically Triads): My friend pays $600 with utilities included (except Wi-Fi and telephone) for an individual room. Otherwise, you can also share a room. 

Both have its advantages and disadvantages, it's up to you to decide. 

Personally, I live in South Campus and I love my room. It's a huge, personal room for $300 with furniture like bed, table, fan, almirah, and wardrobe already provided.
Some of my friends live in Triads and I have visited them. That is not for me.

**Comment by brindashar4** [comment]
Hello! This is kind of a specific question, but I wanted to know if you could shed some light on how Montrose is as a location to get a house at? The house in question is 85 Montrose, so if you could help me get reviews on the house as well as the street, then it would be great!
Thank you!

**Reply by raunaqjain** [reply]
Montrose is a beautiful street, and it has splendid houses too. 85 Montrose is a few metres away from Englewood store, so it's an excellent location.

**Reply by brindashar4** [reply]
Okay, thanks alot Raunaq!

**Comment by Adithyav96** [comment]
Hey! 
Is there anyone who stayed in 150 Springville? Any reviews regarding the street or house can help. Also if anyone has stayed in a house managed by Andy, kindly let me know the pros and cons.

**Comment by abhiraj24** [comment]
Hi,
I have certain queries.It would be really helpful if someone can help me out her as time to leave India is quite near.
- How to get sim and bank account set-up?
- What things to buy a soon as I enter to the apartment?
- What things to carry from India?
